guide/C/ch_invest.xml

```diff
@@ -3151,12 +3199,11 @@ Income
         <para>The term <emphasis>scrub</emphasis> is used because security accounts need to be cleaned after sales
           to ensure the difference between the cost paid for securities, and value received from
           selling them, is accounted for as capital gain or loss. If the capital gain/loss is not
-          correct, the Trial Balance (
-          <menuchoice>
+          correct, the <guimenuitem>Trial Balance</guimenuitem> 
+          (<menuchoice>
```

Comment:
Parentheses are no good delimiter for `<menuchoice>` as they can contain a `<shortcut>` which will also produce a pair of them.

Reply:
ab6c9581